packages/guardian/src/tasks/acalaChain/PoolsTask.ts

```diff
@@ -22,13 +23,16 @@ export default class PoolsTask extends Task<{ currencyId: any }, Pool> {
     let pairs: TradingPair[];
 
     if (currencyId === 'all') {
-      const tradingPair = await apiRx.query.dex.tradingPairStatuses.entries().pipe(take(1)).toPromise();
+      const tradingPair = await apiRx.query.dex.liquidityPool.entries().pipe(take(1)).toPromise();
```

Comment:
`tradingPairStatuses` isn't available on Mandala TC5 but was available on Mandala PC2. I think my work around will work for either version.